src/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -142,36 +139,36 @@ export const PriceOptions: React.FC<PriceOptionsProps> = ({
         ))
         .concat(
           <BorderedRadio
-            id="scrollTo--price-option-custom"
             value="price-option-custom"
             label="Different amount"
             error={showError}
             onSelect={() => {
               customValue && onChange(customValue)
               !toggle && setToggle(true)
             }}
-            key="price-option-custom"
           >
             {toggle && (
-              <Flex flexDirection="column" mt={2}>
-                <OfferInput
-                  id="OfferForm_offerValue"
-                  showError={showError}
-                  onChange={setCustomValue}
-                  onFocus={() => {
-                    onFocus()
-                    scrollTo()
-                  }}
-                  noTitle
-                />
-                {(!customValue || customValue < minPrice) && (
-                  <MinPriceWarning
-                    isPriceRange={!!artwork?.isPriceRange}
-                    minPrice={asCurrency(minPrice) as string}
-                    orderID={order.internalID}
+              <Jump key="price-option-custom" id="price-option-custom">
```

Comment:
**question**: maybe we should pass `key` prop for `BorderedRadio`?

Reply:
Yeah, [RadioGroup](https://github.com/artsy/palette/blob/main/packages/palette/src/elements/RadioGroup/RadioGroup.tsx) is a very weird old component that needs to be revisited — I'll modernize it at some point 🤷